### ISSUE TITLE
Lint/AssignmentInCondition not triggered for property access

### ISSUE THREAD

**Issue by ptarjan** [bbatsov/rubocop#2960]
## Expected behavior

```
if foo.bar = baz
  91
end
```

Should trigger the `Lint/AssignmentInCondition`
## Actual behavior

No error

## Steps to reproduce the problem

```
$ cat a.rb
if foo.bar = baz
  91
end
$ rubocop a.rb
Inspecting 1 file
.

1 file inspected, no offenses detected
```

## RuboCop version

```
$ rubocop -V
0.38.0 (using Parser 2.3.0.6, running on ruby 2.1.2 x86_64-darwin15.0)
```

**Comment by alexdowad** [comment]
I've got a fix.
